core/src/read.jl

```diff
@@ -1722,34 +1722,155 @@ datetime_since(t::Real, t0::DateTime)::DateTime = t0 + Millisecond(round(1000 *
 
 Load a table from a NetCDF file. The data is stored as multi-dimensional arrays, and
 converted to a table for compatibility with the rest of the internals.
+
+Repeat allows us to expand dimensions to make it fit the tabular format.
+Inner repetition is used to add a dimension before the array.
+Outer repetition is used to add a dimension after the array.
+Use nested repeat calls to add multiple dimensions on one side.
```

Comment:
Not sure what this docstring means. It's not a keyword, but something on the internal datastructure?